libglusterfs/src/glusterfs/glusterfs-fops.h

```diff
@@ -225,16 +225,15 @@ typedef enum gf_upcall_flags_t gf_upcall_flags_t;
 
 enum gf_dict_data_type_t {
     GF_DATA_TYPE_UNKNOWN = 0,
-    GF_DATA_TYPE_STR_OLD = 1,
-    GF_DATA_TYPE_INT = 2,
-    GF_DATA_TYPE_UINT = 3,
-    GF_DATA_TYPE_DOUBLE = 4,
-    GF_DATA_TYPE_STR = 5,
-    GF_DATA_TYPE_PTR = 6,
-    GF_DATA_TYPE_GFUUID = 7,
-    GF_DATA_TYPE_IATT = 8,
-    GF_DATA_TYPE_MDATA = 9,
-    GF_DATA_TYPE_MAX = 10,
+    GF_DATA_TYPE_INT = 1,
+    GF_DATA_TYPE_UINT = 2,
+    GF_DATA_TYPE_DOUBLE = 3,
+    GF_DATA_TYPE_STR = 4,
+    GF_DATA_TYPE_PTR = 5,
+    GF_DATA_TYPE_GFUUID = 6,
+    GF_DATA_TYPE_IATT = 7,
+    GF_DATA_TYPE_MDATA = 8,
+    GF_DATA_TYPE_MAX = 9,
```

Comment:
This may have an impact, as its part of protocol, and server (or) client would be still using old version. Can we retain the enum same with that variable removed, but values of other variables remain same?